charts/yurt-manager/templates/yurt-manager-auto-generated.yaml

```diff
@@ -126,6 +126,8 @@ rules:
   resources:
   - nodepools
   verbs:
+  - create
```

Comment:
I think that nodepool create/delete rbac rights are not needed.

Reply:
yes, of course, but the CI / verify was failed if i don't add it, i will try it again.